requestor/src/main/java/de/rwth/imi/flare/requestor/SearchQueryStringBuilder.java

```diff
@@ -79,6 +96,35 @@ private void constructQueryString(){
         appendTimeConstraints();
     }
 
+    private void appendSingleAgeComparison(Double age, Comparator comparator){
+        this.sb.append("birthdate=");
+        switch (comparator.toString()) {
+            case "gt" -> this.sb.append("lt");
+            case "lt" -> this.sb.append("gt");
+            case "ge" -> this.sb.append("le");
+            case "le" -> this.sb.append("ge");
+            case "eq" -> this.sb.append("eq");
+            case "ne" -> this.sb.append("ne");
+        }
+
+        LocalDate dateToCompare = this.timeValueToDate(age);
+        this.sb.append(dateToCompare.toString());
+    }
+
+    private LocalDate timeValueToDate(Double age){
+        int filterValue = age.intValue();
+        LocalDate date = LocalDate.now();
+        switch (this.criterion.getValueFilter().getUnit().getCode()) {
+            case "a" -> date = date.minusYears(filterValue);
+            case "mo" -> date = date.minusMonths(filterValue);
+            case "wk" -> date = date.minusWeeks(filterValue);
+            case "d" -> date = date.minusDays(filterValue);
+            case "h" -> date = date.minusDays(filterValue / 24);
+            case "min" -> date = date.minusDays((filterValue / 60) / 24);
```

Comment:
@bastianschaffer : i think you will have to return an error here "not implemented" - as your conversion makes h and min impossible to query for accurately - this is not what the user would expect